app/presenters/wpcc/message_presenter.rb

```diff
@@ -40,5 +40,18 @@ def above_max_contribution
 
       t('wpcc.contributions.contribution_gt40000_warning', amount: amount)
     end
+
+    def employee_contribution_tip
+      if salary_below_tax_relief_threshold?
```

Comment:
I dont think you want to use this method as it checks for an annual salary equivalent of 11_500